pd/migrations/20211216025603_init.sql

```diff
@@ -33,12 +33,23 @@ CREATE INDEX ON notes (position);
 CREATE INDEX ON notes (height);
 
 CREATE TABLE IF NOT EXISTS validators (
-    tm_pubkey bytea NOT NULL PRIMARY KEY,
-    voting_power bigint NOT NULL
+    tm_pubkey bytea NOT NULL PRIMARY KEY
 );
```

Comment:
non-blocking + non-action, just noting for later: we'll probably want to add in extra metadata fields here, like a display name, website, etc.